daml-lf/tests/BUILD.bazel

```diff
@@ -35,17 +39,17 @@ daml_compile(
     name = "Exceptions",
     srcs = ["Exceptions.daml"],
     # TODO https://github.com/digital-asset/daml/issues/8020
```

Comment:
Probably better if we add a different issue for releasing LF 1.14 (i.e. switching from preview to stable)